app/javascript/components/dashboard-charts-form/areaChart.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
```

Comment:
@kavyanekkalapu wondering if we should use the component name as `dashboard-charts`, forms prefix doesn't make sense as these are not forms.

Reply:
sure, will do